style.css

```diff
@@ -402,3 +421,67 @@ body {
   flex-direction: column;
   text-align: center;
 }
+
+/* CONTACT PART */
+.footer {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  width: 100%;
+  height: auto;
+}
+
+.footer-content {
+  align-items: center;
+  justify-content: center;
+}
+
+.footer-text {
+  padding-top: 3rem;
+}
+
+#best {
+  padding-left: 1.3rem;
+}
+
+#best input {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  width: 18rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  color: #979493;
+  padding: 5px;
+}
+
+#textarea {
+  margin-top: 2rem;
+  background: #fbf8f7;
+}
+
+#best .btn {
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  margin: 2rem 6rem;
+  padding: 10px 10px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+#best .btn:hover {
+  background: rgba(255, 255, 255, 0.24);
+  color: #ff6b00;
+  border: 2px solid #ff6b00;
+}
+
+.down-boarder {
+  padding: 0.5rem 0 0 8rem;
+}
```

Comment:
Your design is still very different to [the Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471): :grimacing: 

- Your design: 
![image](https://user-images.githubusercontent.com/78335447/172971464-e0bf7064-86d0-4dbf-9d03-bf3cc8212e17.png)

Try using the following checklist: :mag: 
- [ ] Remove the extra space you have on the right side (I think it is caused by too large `width`s' in some images or text areas).
- [ ] Make all the fields as wide as the subtitle.
- [ ] Remove the border of your `textarea`
- [ ] Use the correct `color` on your inputs (`#979493`)

Reply:
pls I can't seems to get this part because what I have on my local machine appears aligned and okay .